net-p2p/resilio-sync-bin/resilio-sync-bin-2.5.6.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit pax-utils systemd tmpfiles unpacker user
+
+MY_PN="${PN/-bin/}"
+BASE_URI="http://linux-packages.resilio.com/${MY_PN}/deb/pool/non-free/r/${MY_PN}/${MY_PN}_${PV}-1_-arch-.deb"
```

Comment:
Prefer using uncommon characters for substitutions to make them stand out better, i.e. `@arch@`.